flag/service/guest/guest.go

```diff
@@ -0,0 +1,7 @@
+package guest
+
+import "github.com/giantswarm/aws-operator/flag/service/guest/update"
```

Comment:
All this configuration is basically copy/paste from the kvm-operator, where we configure updates already the same way.